test/run_test.py

```diff
@@ -370,7 +376,7 @@ def exclude_tests(exclude_list, selected_tests, exclude_message=None):
     tests_copy = selected_tests[:]
     for exclude_test in exclude_list:
         for test in tests_copy:
-            if test.startswith(exclude_test):
+            if test == exclude_test:
```

Comment:
why changing this to exact match?

Reply:
Notice line 85,

```py
WINDOWS_BLACKLIST = [
    'distributed',  # line 85
    'distributed/rpc/process_group/rpc_fork',
]
```


Initially `selected_tests` is like,
```py
selected_tests = [
    'distributed',
    'distributed/rpc/process_group/rpc_fork',
]
```

After iterating on `exclude_test == 'distributed'`, `selected_tests` becomes,

```py
selected_tests = [
    'distributed/rpc/process_group/rpc_fork',
]
```

Since the `exclude_tests` is,
```py
def exclude_tests(exclude_list, selected_tests, exclude_message=None):
    tests_copy = selected_tests[:]
    for exclude_test in exclude_list:
        for test in tests_copy:
            if test.startswith(exclude_test):
                if exclude_message is not None:
                    print_to_stderr('Excluding {} {}'.format(test, exclude_message))
                selected_tests.remove(test)
    return selected_tests
````

when 
`test == 'distributed/rpc/process_group/rpc_fork'`,
and
`exclude_test == 'distributed'`

It will run `selected_tests.remove('distributed')`, raising a "list does not contain "'distributed'"" error.